src/helpers/styler.js

```diff
@@ -69,10 +71,12 @@ export function morphDraggedElementToBeLike(draggedEl, copyFromEl, currentMouseX
     const computedStyle = window.getComputedStyle(copyFromEl);
     Array.from(computedStyle)
         .filter(s => s.startsWith('background') || s.startsWith('padding') || s.startsWith('font') || s.startsWith('text') || s.startsWith('align') ||
-        s.startsWith('justify') || s.startsWith('display') || s.startsWith('flex') || s.startsWith('border') || s === 'opacity' || s === 'color')
+            s.startsWith('justify') || s.startsWith('display') || s.startsWith('flex') || s.startsWith('border') || s === 'opacity' || s === 'color')
         .forEach(s =>
             draggedEl.style.setProperty(s, computedStyle.getPropertyValue(s), computedStyle.getPropertyPriority(s))
         );
+
+    transformDraggedElement(draggedEl);
```

Comment:
no need to pass in the draggedEl right? it is a function that takes no params (curried)
`transformDraggedElement()` should be fine

Reply:
Woops left that from an earlier test, fixed.